content/chat/rooms/reactions.textile

```diff
@@ -16,16 +17,31 @@ Room reactions are ephemeral and not stored or aggregated by Ably. The intention
 
 h2(#subscribe). Subscribe to room reactions
 
-Subscribe to room reactions by registering a listener. These events are emitted whenever a user sends a reaction, such as by hitting a 'like' button or clicking a heart emoji.
+blang[javascript].
+  Subscribe to room reactions by registering a listener. These events are emitted whenever a user sends a reaction, such as by hitting a 'like' button or clicking a heart emoji. Use the "@reactions.subscribe()@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat_js.RoomReactions.html#subscribe method in a room to receive reactions:
 
-Use the "@reactions.subscribe()@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat_js.RoomReactions.html#subscribe method in a room to receive reactions:
+blang[react].
+  Subscribe to room reactions with the "@useRoomReactions@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/functions/chat_react.useRoomReactions.html hook. Supply an optional listener to receive the room reactions.
 
 ```[javascript]
 const {unsubscribe} = room.reactions.subscribe((reaction) => {
   console.log(`Received a reaction of type ${reaction.type}, and metadata ${reaction.metadata}`);
 });
 ```
 
+```[react]
+import React, { useCallback } from 'react';
+import { useRoomReactions } from '@ably/chat/react';
+
+const MyComponent = () => {
+  const { send } = useRoomReactions({
```

Comment:
If we are showing the use of send, can we do the same as with `useMessages` and give an example button with params please? :)